joerd/command.py

```diff
@@ -173,6 +207,98 @@ def joerd_process(cfg):
     j.process()
 
 
+def joerd_server(global_cfg):
+    assert cfg.sqs_queue_name is not None, \
+        "Could not find SQS queue name in config, but this must be configured."
+
+    sqs = boto3.resource('sqs')
+    queue = sqs.Queue(cfg.sqs_queue_name)
+
+    while True:
+        for message in queue.get_messages():
+            region = json.loads(message.body)
+            cfg = global_cfg.copy_with_regions([region])
+            joerd_process(cfg)
```

Comment:
I thought we'd have an auto-scaling group with a rule like "if > 20 jobs in queue then keep the pool at max, if the queue is at zero, scale it down to one instance" or something, and that would be good enough for a repeated "one-off" run... What do you think? We could put some signal handling stuff in there to make shutdown nicer, but I've read that it doesn't interact nicely with multiprocessing.